flink-libraries/flink-streaming-python/src/test/java/org/apache/flink/streaming/python/api/PythonStreamBinderTest.java

```diff
@@ -63,7 +63,7 @@ private static Path findUtilsModule() {
 
 	@Test
 	public void testProgram() throws Exception {
-		Path testEntryPoint = new Path(getBaseTestPythonDir(), "examples/word_count.py");
+		Path testEntryPoint = new Path(getBaseTestPythonDir(), "run_all_tests.py");
```

Comment:
Is this change relevant?

Reply:
Basically this change actually enables most python tests, which i accidentally disabled months ago. So yeah, to enable more tests we do need this.